input/templates/partials/myaccount/my-account-sidebar.hbs

```diff
@@ -1,29 +1,29 @@
   </div>
-  <div {{#if positionTwo}}id="my-account-mobile-content"{{/if}}class="my-account-sidebar-items {{#if positionTwo}}active{{/if}}">
+  <div {{#if changeMyPasswordTab}}id="my-account-mobile-content"{{/if}}class="my-account-sidebar-items {{#if changeMyPasswordTab}}active{{/if}}">
+    <a href="{{@root.meta._links.changePassword.href}}">{{i18n "my-account:myAccountSidebar.changePassword"}}</a>
+  </div>
+  <div {{#if myAddressBookTab}}id="my-account-mobile-content"{{/if}}class="my-account-sidebar-items {{#if myAddressBookTab}}active{{/if}}">
     <a href="{{@root.meta._links.myAddressBook.href}}">{{i18n "my-account:myAccountSidebar.addressBook"}}</a>
   </div>
-  <div {{#if positionThree}}id="my-account-mobile-content"{{/if}}class="my-account-sidebar-items {{#if positionThree}}active{{/if}}">
+  <div {{#if myPaymentDetailsTab}}id="my-account-mobile-content"{{/if}}class="my-account-sidebar-items {{#if myPaymentDetailsTab}}active{{/if}}">
     <a href="{{@root.meta._links.myPaymentDetails.href}}">{{i18n "my-account:myAccountSidebar.paymentDetails"}}</a>
   </div>
-  <div {{#if positionFour}}id="my-account-mobile-content"{{/if}}class="my-account-sidebar-items {{#if positionFour}}active{{/if}}">
+  <div {{#if myOrdersTab}}id="my-account-mobile-content"{{/if}}class="my-account-sidebar-items {{#if myOrdersTab}}active{{/if}}">
     <a href="{{@root.meta._links.myOrders.href}}">{{i18n "my-account:myAccountSidebar.myOrders"}}</a>
   </div>
-  <div {{#if positionFive}}id="my-account-mobile-content"{{/if}}class="my-account-sidebar-items {{#if positionFive}}active{{/if}}">
+  <div {{#if myReturnsAndExchangesTab}}id="my-account-mobile-content"{{/if}}class="my-account-sidebar-items {{#if myReturnsAndExchangesTab}}active{{/if}}">
     <a href="{{@root.meta._links.myReturnsAndExchanges.href}}">{{i18n "my-account:myAccountSidebar.returnsExchange"}}</a>
   </div>
-  <div {{#if positionSix}}id="my-account-mobile-content"{{/if}}class="my-account-sidebar-items clearfix {{#if positionSix}}active{{/if}}">
+  <div {{#if myWishlistTab}}id="my-account-mobile-content"{{/if}}class="my-account-sidebar-items clearfix {{#if myWishlistTab}}active{{/if}}">
     <a href="{{@root.meta._links.myWishlist.href}}">
       <span><img class="sidebar-wishlist-heart hidden-sm" src="{{meta.assetsPath}}/img/heart-1.png" alt="heart icon"></span>{{i18n "my-account:myAccountSidebar.wishlist"}}<span class="sidebar-wishlist-quantity hidden-sm">{{content.sidebarItemSix.wishlistItems}}</span>
     </a>
   </div>
-  <div {{#if positionSeven}}id="my-account-mobile-content"{{/if}}class="my-account-sidebar-items {{#if positionSeven}}active{{/if}}">
-    <a href="{{@root.meta._links.changePassword.href}}">{{i18n "my-account:myAccountSidebar.changePassword"}}</a>
-  </div>
-  <div {{#if positionEight}}id="my-account-mobile-content"{{/if}}class="my-account-sidebar-items {{#if positionEight}}active{{/if}}">
```

Comment:
I have removed all tab selector from sign out, since it will never be selected I guess ;)